### PR TITLE
Add gitignore, use github actions for automagical complication and add readme

### PR DESCRIPTION
**Pull request by Silverdimond**

### CODE REVIEW

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+         tag_name: ${{ steps.E.outputs.Random }}
+         release_name: ${{ steps.E.outputs.Random }}
+         body: Automated release, a release will be created for every single push on the master branch. this one is for ${{ github.sha }}
+         draft: true
```
Comment (Silverdimond) [inline .github/workflows/dotnet.yml:39]: You might want to change this and line 40

Comment (catdotjs) [reply .github/workflows/dotnet.yml:39]: idk what this even does

**Review by catdotjs — APPROVED** [review]
cool

**Review by catdotjs — APPROVED** [review]
ye i guess it works

**Pull request merged**